### PR TITLE
Project management: Add step that updates CHANGELOG files before npm releases

### PR DESCRIPTION
**Pull request by gziolo**
## Description

This PR renames `./bin/commander.js npm-stable` to `./bin/commander.js npm-stable-minor` to better reflect the actual use case.

In addition and more importantly, it automates the process of updating CHANGELOG files during the process of publishing packages to npm. Docs were updated to reflect the refined workflow:

https://github.com/WordPress/gutenberg/blob/a66eb217e61c9d9fceb2fe1df8609b0e2a3357fc/docs/contributors/release.md#synchronizing-wordpress-trunk

## How has this been tested?
I testes all steps but the one that push to the origin. You can simulate it with the following diff:

```diff
diff --git a/bin/commander.js b/bin/commander.js
index 799aa901c..1746d58dc 100755
--- a/bin/commander.js
+++ b/bin/commander.js
@@ -853,10 +853,10 @@ async function prepublishPackages( minimumVersionBump ) {
 
 	// Push the local changes
 	abortMessage = `Aborting! Make sure to push changes applied to WordPress release branch "${ releaseBranch }" manually.`;
-	await runPushGitChangesStep( releaseBranch, abortMessage );
+	//await runPushGitChangesStep( releaseBranch, abortMessage );
 
 	abortMessage = 'Aborting! The release is finished though.';
-	await runCleanLocalCloneStep( abortMessage );
+	//await runCleanLocalCloneStep( abortMessage );
 }
 
 program
```
The full workflow should be tested with the actual release :)

## Screenshots 

Screen Shot 2020-01-20 at 14 48 38
Screen Shot 2020-01-20 at 15 21 14

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR. .

### CODE REVIEW

File: bin/commander.js [hunk 1]
```diff
@@ -754,17 +860,17 @@ async function prepublishPackages() {
 }
 
 program
-	.command( 'prepublish-packages-stable' )
-	.alias( 'npm-stable' )
-	.description( 'Prepublish to npm steps for a stable version of WordPress packages' )
+	.command( 'prepublish-packages-stable-minor' )
+	.alias( 'npm-stable-minor' )
```
Comment (youknowriad) [inline bin/commander.js]: I'm not sure I understand why this is a "minor"?

Comment (gziolo) [reply bin/commander.js]: there needs to be at least `minor` version bump when syncing trunk according to the existing docs. Did I miss something?

Comment (gziolo) [reply bin/commander.js]: I think we can leave it as `npm-stable` and add an argument option to the command when we add handling for patch releases.

Comment (youknowriad) [reply bin/commander.js]: Oh got it 👍 Yeah I think that should be an argument probably

Comment (gziolo) [reply bin/commander.js]: Cool, updated with 9791c81.

**Review by youknowriad — APPROVED** [review]
This is cool, let's give it a shot.

File: bin/commander.js [hunk 2]
```diff
@@ -729,12 +731,114 @@ async function runWordPressReleaseBranchSyncStep( abortMessage ) {
...
+			} )
+	);
+
+	await askForConfirmationToContinue(
+		`All corresponding files were updated. Commit the changes?`,
+		true,
+		abortMessage
+	);
+	const simpleGit = SimpleGit( gitWorkingDirectoryPath );
+	await simpleGit.add( './*' );
```
Comment (mcsf) [inline bin/commander.js:829]: @gziolo: I'm only five and a half years late 😄, but was there any difference between git-adding `./*` and `.`?

Comment (gziolo) [reply bin/commander.js:829]: I don't remember 😅 I have no idea whether it makes any difference at this point.

Comment (mcsf) [reply bin/commander.js:829]: I noticed that it's one of those values that has been propagated through the changes, so there's a risk of cargo-cult programming. But it's fine, we're talking about a minor thing! 😅

**Pull request merged**